templates/node--16.html.twig

```diff
@@ -1,5 +1,3 @@
-{{ attach_library('eirh_subtheme/eventshub') }}
```

Comment:
In theory, with the library loaded on the alias `/resources` we could remove the node template file and put the `<div id="su-eventshub">Please view this page in another browser.</div>` in an embed block somewhere.